app/bundles/ReportBundle/Model/ReportModel.php

```diff
@@ -733,21 +733,30 @@ private function getOperatorOptions(array $data)
     }
 
     /**
-     * @return int
+     * Get the total of rows found by the query. Ignore pagination limits.
      */
-    private function getTotalCount(QueryBuilder $qb, array &$debugData)
+    private function getTotalCount(QueryBuilder $qb, array &$debugData): int
     {
         $countQb = clone $qb;
-        $countQb->resetQueryParts();
 
-        $countQb->select('count(*)')
-            ->from('('.$qb->getSQL().')', 'c');
+        // only COUNT the group by field(s) to comply with SQLs only_full_group_by mode
+        $groupBy = $qb->getQueryPart('groupBy');
+        if(!is_array($groupBy)){
+            throw new \Exception('A group by statement is required in order to get the total rows');
```

Comment:
Are you sure this exception is caught and nicely presented to the user? I don't see it being caught anywhere. It should also be translatable.

Reply:
as this is an unlikely case I dont think we need to present it to the user?  It is enough if it goes to the logs?

Reply:
How about this:
1. Create a report based on the Contacts data point.
3. Add the email address as a column of the report.
4. Save&Close the report

There is no group by in such report query. So there should be a 500 error in this case?